client/src/app/account-card/account-card.component.html

```diff
@@ -0,0 +1,11 @@
+<aside class="account-container">
+  <mat-card class="account-card">
+    <mat-card-subtitle class="account-card__title">{{
```

Comment:
{{ account.title | titlecase }} 
write interpolation in one line, please
in every places